### ISSUE TITLE
RIP - maxtrajtime

### ISSUE THREAD

**Issue by fherreazcue** [UoMResearchIT/SpanishPlumeAnalysis#9]
RIP throws an error if the back-trajectories being calculated span for over 33 hours. 

The reason is a variable in RIP called maxtrajtime, that is set (for some unknown reason) to 200. This seems to be the maximum number of iterations for the trajectory computation.
Current trajectories use a dt of 600s, and so 200*600s=33.33h.

We need to find out how to change maxtrajtime.

**Comment by douglowe** [comment]
`maxtrajtime` is set as a parameter within the fortran code, so we will have to hard-code any changes, and rebuild the RIP docker image.

And, helpfully, it is hard coded in several places (rather than using one central point of truth...).

The locations in which it is set are:
```
src/htrajdraw.f:24:      parameter (maxtraj=7000,maxtrajtime=200)
src/netasc.f:32:      parameter (maxtraj=7000,maxtrajtime=200)
src/showtraj.f:3:      parameter (maxtraj=7000,maxtrajtime=200)
src/tabdiag.f:3:      parameter (maxtraj=7000,maxtrajtime=200,maxvar=20)
src/vtrajdraw.f:21:      parameter (maxtraj=7000,maxtrajtime=2800)
```
Strangely it is set to 2800 in `vtrajdraw.f`, but 200 everywhere else. I don't think we need a limit of 2800(?), so perhaps we go for 400 or 600 instead.

Can you add `sed` commands to the docker build script - similar to those already included there? Use the search string `maxtrajtime=200`, and replace it with the appropriate value.

**Comment by fherreazcue** [comment]
Resolved in [1c916ad](https://github.com/UoMResearchIT/RIP-docker/commit/1c916ad22df416db083b84f8ead8fdcc97798345).

**Comment by fherreazcue** [comment]
Results shared through [Dropbox](https://www.dropbox.com/home/Spanish%20Plume/2023-12_Trajectories_80?di=left_nav_browse)
